apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -2,28 +2,35 @@ import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
   id: 'auth-sso-saml',
-  title: 'Single Sign-On with SAML 2.0',
-  description: 'Use Single Sign-On (SSO) authentication with SAML 2.0',
+  title: 'Single Sign-On with SAML 2.0 for Projects',
+  description: 'Use Single Sign-On (SSO) authentication on your project with SAML 2.0',
   video: 'https://www.youtube.com/v/em1cpOAXknM',
 }
 
-Supabase Auth supports enterprise-level Single Sign-On (SSO) for any identity providers compatible with the using the SAML 2.0 protocol.
+Supabase Auth supports enterprise-level Single Sign-On (SSO) for any identity providers compatible with the using the SAML 2.0 protocol. This is a non-exclusive list of supported identity providers:
 
-<Admonition type="info">
-This is an early beta release of these APIs. CLI and Dashboard support for SSO is under development.
+- Google Workspaces (formerly known as GSuite)
+- Okta, Auth0
+- Microsoft Active Directory, Azure Active Directory, Microsoft Entra
+- PingIdentity
+- OneLogin
 
-If you are comfortable using these APIs and would like to try out SSO with SAML 2.0 for your project, please [open a support ticket](https://app.supabase.com/support/new).
+If you're having issues with identity provider software not on this list, please [open a support ticket](https://app.supabase.com/support/new).
 
-These APIs are not expected to change before the feature is generally available, but we do reserve the right to modify them. Projects in the Beta will be notified of any changes.
+## Prerequisites
 
-</Admonition>
+This guide requires the use of the [Supabase CLI](/docs/guides/cli). Please make sure you're using version v1.46.0 or higher. You can use `supabase -v` to see the currently installed version.
```

Comment:
```suggestion
This guide requires the use of the [Supabase CLI](/docs/guides/cli). Please make sure you're using version v1.46.4 or higher. You can use `supabase -v` to see the currently installed version.
```
Probably should update this once dogfooding is done, guessing that it might be v1.46.4 after any patches